packages/cycle-scripts/package.json

```diff
@@ -30,12 +30,16 @@
     "cycle-scripts": "./index.js"
   },
   "dependencies": {
-    "babel-core": "^6.24.1",
```

Comment:
no, because `^6.24.1` will match every minor and patch update. So the dependencies of the blocks are automaticly kept up to date, unless there is a new major version
This is what `npm udate` is for